presto-main/src/main/java/com/facebook/presto/operator/aggregation/AggregationUtils.java

```diff
@@ -88,14 +88,14 @@ public static void updateCorrelationState(CorrelationState state, double x, doub
 
     public static double getCorrelation(CorrelationState state)
     {
-        // Math comes from ISO9075-2:2011(E) 10.9 General Rules 7 c x
```

Comment:
Nice catch! This looks correct. 

One comment I have is I find it easier to follow this code if we calculate it with the exact definition of
`CORR = COVAR_POP(expr1, expr2) / (STDDEV_POP(expr1) * STDDEV_POP(expr2))`
 instead of simplifying this expression by multiplying all the terms with `state.getCount()`.



```
double covariancePopulation = getCovariancePopulation(state);
double stdDevPopulationX = Math.sqrt((state.getSumXSquare() - state.getSumX() * state.getSumX() / state.getCount()) / state.getCount());
double stdDevPopulationY = Math.sqrt((state.getSumYSquare() - state.getSumY() * state.getSumY() / state.getCount()) / state.getCount());
 // stdDevPopulationX and stdDevPopulationY deliberately not checked for zero because the result can be Infinity or NaN even
 // if they are both not zero
return covariancePopulation / (stdDevPopulationX * stdDevPopulationY);
```